configs/base_env.rc

```diff
@@ -13,7 +13,7 @@ smtp_host = localhost
 imap_host = localhost
 piwik_host = localhost
 *sentry_scheme = https
-*sentry_host = localhost
+*sentry_host = sentry.bluenove.com
```

Comment:
Anything bluenove-specific should go in `configs/bluenove-server-configs`, in this case `configs/bluenove-server-configs/bluenove.rc`

Reply:
The key `*sentry_host` is already set in `configs/bluenove-server-config/bluenove.rc`. In `configs/base_env.rc`, should I set it back to `localhost` or should I remove the key?